android/java/org/chromium/chrome/browser/appmenu/BraveShieldsMenuAdapter.java

```diff
@@ -265,7 +266,7 @@ public View getView(int position, View convertView, ViewGroup parent) {
                         number.setTag(R.string.brave_shields_scripts_blocked);
                     }
                 } else if (6 == position) {
-                    convertView = mInflater.inflate(R.layout.brave_shields_menu_item, parent, false);
+//                    convertView = mInflater.inflate(R.layout.brave_shields_menu_item, parent, false);
```

Comment:
why are these commented out?

Reply:
These are commented out just to make it build. There's a follow up issue for the Android team to fix:
brave/brave-browser#7726